test/e2e/specs-playwright/wp-log-in-out-spec.js

```diff
@@ -1,32 +1,50 @@
 /**
  * External dependencies
  */
+import config from 'config';
 
-import playwright from 'playwright';
+/**
+ * Internal dependencies
+ */
+import * as browserManager from '../dist/lib/browser-manager';
```

Comment:
Was this supposed to import from `dist` or directly from `./lib/browser-manager`?

Reply:
My understanding (correct me if I'm wrong) is that plain JS file cannot import TS. I implemented a workflow that requires compilation to JS which is output into the `dist/` directory, and that is imported in the tests.

Reply:
Ah, plain JS should be able to import TypeScript just fine as we use the babel parser for TypeScript. See https://github.com/automattic/wp-calypso/blob/907b61456eab841f0941e5ee1e4f302b339d0ecb/client/sections-middleware.js#L18 for an example (`add-reducer` is TypeScript).

Reply:
This is meant to be run by Node. I think Mocha uses [esm](https://github.com/standard-things/esm) to bring in support for ESM imports without transpiling the code first with babel.

At least in my local tests this has to point to `../dist/lib/browser-manager`, otherwise Mocha+esm won't be able to find it.

Reply:
@scinos I have a follow-up PR where I was able to load TypeScript into plain JavaScript test files using `ts-node`, if you'd like to take a look. https://github.com/Automattic/wp-calypso/pull/51419